src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsRepeatableIncontentBoxadMobileWiki.java

```diff
@@ -53,18 +53,32 @@ private static String urlWithInstantGlobals(boolean enabled) {
   public void adDisplayedBeforeSections() {
     AdsBaseObject ads = new AdsBaseObject();
     ads.getUrl(urlWithInstantGlobals(true));
-    for (int i = 1; i <= HEADERS_WITH_AD_NUMBER; i++) {
-      ads.scrollTo(By.id(String.format(ARTICLE_HEADER_SELECTOR, Integer.toString(i))));
 
-      Assertion.assertTrue(isIncontenAdDisplayed(i, ads),
-                           "IncontentBoxad is not displayed before section"
-      );
-    }
-    ads.scrollTo(By.id(String.format(ARTICLE_HEADER_SELECTOR, Integer.toString(HEADERS_WITH_AD_NUMBER + 1))));
 
-    Assertion.assertFalse(isIncontenAdDisplayed(HEADERS_WITH_AD_NUMBER + 1, ads),
-                            "IncontentBoxad is displayed before section"
-      );
+    ads.scrollTo(By.id(String.format(ARTICLE_HEADER_SELECTOR, Integer.toString(1))));
+    Assertion.assertTrue(isIncontenAdDisplayed(1, ads),
+                         "IncontentBoxad is not displayed before section"
+    );
+    ads.scrollTo(By.id(String.format(ARTICLE_HEADER_SELECTOR, Integer.toString(2))));
+    Assertion.assertFalse(isIncontenAdDisplayed(3, ads),
+                          "IncontentBoxad is displayed before section"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "IncontentBoxad is displayed before section" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)